### ISSUE TITLE
Corrupted STDOUT by directly writing to native stream in forked JVM 1

### ISSUE THREAD

**Issue by straumat** [qase-tms/qase-java#13]
I added Qase to my Java project [Cassandre](https://github.com/cassandre-tech/cassandre-trading-bot).

When running the tests with Maven, I get an error:
```
[WARNING] Corrupted STDOUT by directly writing to native stream in forked JVM 1. See FAQ web page and the dump file /home/straumat/IdeaProjects/cassandre-trading-bot/spring-boot-starter/autoconfigure/target/surefire-reports/2021-01-18T15-28-03_010-jvmRun1.dumpstream
```

The reports says : 
```
# Created at 2021-01-18T15:28:03.901
Corrupted STDOUT by directly writing to native stream in forked JVM 1. Stream '15:28:03.895 [main] INFO io.qase.junit5.QaseExtension - Qase project code - CTB'.
java.lang.IllegalArgumentException: Stream stdin corrupted. Expected comma after third character in command '15:28:03.895 [main] INFO io.qase.junit5.QaseExtension - Qase project code - CTB'.
	at org.apache.maven.plugin.surefire.booterclient.output.ForkClient$OperationalData.<init>(ForkClient.java:507)
	at org.apache.maven.plugin.surefire.booterclient.output.ForkClient.processLine(ForkClient.java:210)
	at org.apache.maven.plugin.surefire.booterclient.output.ForkClient.consumeLine(ForkClient.java:177)
	at org.apache.maven.plugin.surefire.booterclient.output.ThreadedStreamConsumer$Pumper.run(ThreadedStreamConsumer.java:88)
	at java.base/java.lang.Thread.run(Thread.java:834)


# Created at 2021-01-18T15:28:03.903
Corrupted STDOUT by directly writing to native stream in forked JVM 1. Stream '15:28:03.898 [main] INFO io.qase.junit5.QaseExtension - Qase run id - 123'.
java.lang.IllegalArgumentException: Stream stdin corrupted. Expected comma after third character in command '15:28:03.898 [main] INFO io.qase.junit5.QaseExtension - Qase run id - 123'.
	at org.apache.maven.plugin.surefire.booterclient.output.ForkClient$OperationalData.<init>(ForkClient.java:507)
	at org.apache.maven.plugin.surefire.booterclient.output.ForkClient.processLine(ForkClient.java:210)
	at org.apache.maven.plugin.surefire.booterclient.output.ForkClient.consumeLine(ForkClient.java:177)
	at org.apache.maven.plugin.surefire.booterclient.output.ThreadedStreamConsumer$Pumper.run(ThreadedStreamConsumer.java:88)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Note that I'm forced to run my tests with the option for `maven-surefire-plugin:`
```
<reuseForks>false</reuseForks>
<forkCount>1</forkCount>
```

I don't see any result posted in Qase.

**Comment by straumat** [comment]
The results are in fact correctly posted to Qase

**Comment by savkk** [comment]
This issue has been fixed in #24. If the problem  is reproduced, please let me know.
